src/spikeinterface/postprocessing/tests/common_extension_tests.py

```diff
@@ -196,9 +234,9 @@ def test_extension(self):
         # test content of memory/content/zarr
         for ext in self.we2.get_available_extension_names():
             print(f"Testing data for {ext}")
-            ext_memory = self.we2.load_extension(ext)
+            ext_memory = self.we_memory2.load_extension(ext)
             ext_folder = self.we2.load_extension(ext)
-            ext_zarr = self.we2.load_extension(ext)
+            ext_zarr = self.we_zarr2.load_extension(ext)
```

Comment:
@alejoe91 : look at theses two lines. It was a bug in the test suite we were comparing the same 3 times.
I change it but now it fails because some extension need randoms and do not give exatly the same results for instance noise level.